webdriver/tests/bidi/script/evaluate/serialization.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+from webdriver.bidi.modules.script import ContextTarget
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize(
+    "expression, expected",
+    [
+        ("undefined", {"type": "undefined"}),
+        ("null", {"type": "null"}),
+        ("'foobar'", {"type": "string", "value": "foobar"}),
+        ("Number('a')", {"type": "number", "value": "NaN"}),
+        ("-0", {"type": "number", "value": "-0"}),
+        ("Infinity", {"type": "number", "value": "+Infinity"}),
+        ("-Infinity", {"type": "number", "value": "-Infinity"}),
+        ("1 + 2", {"type": "number", "value": 3}),
+        ("1 + 0.4", {"type": "number", "value": 1.4}),
+        ("1 === 1", {"type": "boolean", "value": True}),
+        ("1 !== 1", {"type": "boolean", "value": False}),
```

Comment:
Sorry for not being clear enough. I meant simplification in other lines as well. I guess it's way more readable. Not a blocker though.
```suggestion
        ("3", {"type": "number", "value": 3}),
        ("1.4", {"type": "number", "value": 1.4}),
        ("true", {"type": "boolean", "value": True}),
        ("false", {"type": "boolean", "value": False}),
```

Reply:
Yeah, you're right. Especially since it's only about serialization now, it can be simpler. 
I've updated it.